client/signup/site-mockup/index.jsx

```diff
@@ -88,7 +85,7 @@ class SiteMockups extends Component {
 			this.props.recordTracksEvent( 'calypso_signup_site_preview_mockup_rendered', {
 				site_type: this.props.siteType,
 				vertical_slug: this.props.verticalSlug,
-				site_style: this.props.siteStyle || 'default',
```

Comment:
I think we can just stop recording `site_style: 'default'` as it's no longer meaningful

Reply:
the same pattern could be removed at line `129`